games/CoinFlip.py

```diff
@@ -28,13 +34,20 @@ def get_player_observation(self, player_id):
 
         """
         return {
-            "moves": ['h', 't'],
-            "observation": [],
+            "moves": ["h", "t"],
+            "observation": {
+                "h": "{0:.0%}".format(
```

Comment:
One feature we could add, is to allow players watch other players take their turns (if the game rules allows that). That would allow a clever player to keep track of these kind of statistics.